### DISCUSSION TITLE
where clause with supabase

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#9282] [Questions] [answered]
Hi, I'm trying to get the query to return dates from today onwards. Is there a filter that allows it to be done with supabase as a "where".
<img width="355" alt="Captura de Pantalla 2022-10-02 a la(s) 13 30 42" src="https://user-images.githubusercontent.com/36648562/193465262-110fdc33-dcc8-4821-8fa0-b445bb39e661.png">

**Comment by GaryAustin1** [comment, marked as answer]
You just use .gte,.gt,.lt,.lte as appropriate.

https://github.com/supabase/supabase/discussions/1892#discussioncomment-897341

You just need to figure out exactly what time you want to run the filter on and adjust the date/time you send.  But it is not clear what from "today onwards" means.

**Reply by ricardosikic** [reply]
Hey it worked i had to use `gte()` filter and format the date. Thanks!

` const [todayDate] = new Date(Date.now()).toISOString().split('T')
    try {
      let { data:newData, error:er } = await supabase
      .from('events')
      .select('title')
      .gte('start_date', todayDate)
`
